internal/routing/release.go

```diff
@@ -1,6 +1,7 @@
 package routing
```

Comment:
This follows the suite of comments as for `acquire.go`.

Reply:
I don't understand? Could you elaborate?